polymesh_schema.json

```diff
@@ -1356,6 +1334,27 @@
           }
         ],
         "type": "Option<KeyIdentityData>"
+      },
+      "validCDDClaims": {
+        "description": "Returns all valid IdentityClaim of type CustomerDueDiligence for the given target_identity",
+        "params": [
+          {
+            "name": "target_identity",
+            "type": "IdentityId",
+            "isOptional": false
+          },
+          {
+            "name": "cdd_checker_leeway",
+            "type": "u64",
+            "isOptional": true
+          },
+          {
+            "name": "blockHash",
+            "type": "Hash",
+            "isOptional": true
+          }
```

Comment:
I think there is an issue which prevents us for using multiple `isOptional: true`.  Which is why the other RPC endpoints use `Option<T>` and `isOptional: false`.